coast/TIDEGAUGE.py

```diff
@@ -824,30 +812,30 @@ def read_bodc_data(fn_bodc, date_start=None, date_end=None, header_length: int =
                 if line_count > header_length:
                     try:
                         working_line = line.split()
-                        time_str = working_line[1] + " " + working_line[2]  # Empty lines cause trouble
+                        time_str = working_line[1] + ' ' + working_line[2]  # Empty lines cause trouble
                         sea_level_str = working_line[3]
                         residual_str = working_line[4]
                         if sea_level_str[-1].isalpha():
                             qc_flag_str = sea_level_str[-1]
-                            sea_level_str = sea_level_str.replace(qc_flag_str, "")
-                            residual_str = residual_str.replace(qc_flag_str, "")
+                            sea_level_str = sea_level_str.replace(qc_flag_str, '')
+                            residual_str = residual_str.replace(qc_flag_str, '')
                         elif residual_str[-1].isalpha():  # sometimes residual has a
                             # flag when elevation does not
                             qc_flag_str = residual_str[-1]
-                            sea_level_str = sea_level_str.replace(qc_flag_str, "")
-                            residual_str = residual_str.replace(qc_flag_str, "")
+                            sea_level_str = sea_level_str.replace(qc_flag_str, '')
+                            residual_str = residual_str.replace(qc_flag_str, '')
                         else:
-                            qc_flag_str = ""
-                        # print(line_count-header_length, residual_str, float(residual_str))
-                        # print(working_line, sea_level_str, qc_flag_str)
+                            qc_flag_str = ''
+                        # debug(line_count-header_length, residual_str, float(residual_str))
+                        # debug(working_line, sea_level_str, qc_flag_str)
                         time.append(time_str)
                         qc_flags.append(qc_flag_str)
                         sea_level.append(float(sea_level_str))
                         residual.append(float(residual_str))
                     except:
```

Comment:
We should avoid using bare excepts. These will catch **everything**. Including system exits.